### DISCUSSION TITLE
edit message log

### DISCUSSION THREAD

**Discussion by Catta1997** [Pycord-Development/pycord#2356] [Q&A] [answered]
Hi all, I have some (small) issue with the edit event. I tried the given example and another found. My code are atm:
```
@bot.event
async def on_message_edit(message_before, message_after):
    embed = discord.Embed(title="{} edited a message".format(message_before.author.name),
                          description="", color=0xFF0000)
    embed.add_field(name=message_before.content, value="This is the message before any edit",
                    inline=True)
    embed.add_field(name=message_after.content, value="This is the message after the edit",
                    inline=True)
    channel = bot.get_channel(myChannelLogID)
    await channel.send(channel, embed=embed)
```
It works but the only issue is that the bot log as edit all the gif, the 2 message are the same but log as edited (maybe since discord "show" the url as gif and this trigger the even. any way to disable this specific event?
Thanks

**Comment by plun1331** [comment, marked as answer]
You should probably just compare the contents to see if they're the same, as this also happens when someone removes attachments or embeds from a message.
